qiime2/sdk/plugin_manager.py

```diff
@@ -11,6 +11,8 @@
 import pkg_resources
 import qiime2.core.type
 
+from enum import Flag, auto
```

Comment:
Can you change this to

```suggestion
import enum
```

then, update the code below as appropritate

Reply:
Can `Flag` and `auto` be removed? I thought those are necessary due to their use in the `FormatFilter` class that is created on line `147`?

Reply:
No, they can't be removed. You can access them through the `enum` package like this: `enum.Flag`, `enum.auto`.